check-version-bump.sh

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env bash
+
+VERSION_CHANGE_TRIGGERS="setup.py MANIFEST.in openfisca_france"
+
+current_version=`python setup.py --version`
+
+if git diff-index --quiet origin/master $VERSION_CHANGE_TRIGGERS ":(exclude)*.md"
+then exit 0  # there are no functional changes at all, the version is correct
+fi
+
+if git rev-parse --verify --quiet $current_version
+then
+    echo "Version $current_version already exists:"
+    git log -1 $current_version
```

Comment:
On commit e9a7f82201a85e85d3bf43df61114ca857d7ad72, with an empty `git status`, I ran `./check-version-bump.sh` and I locked on the `git log` screen.